src/Impostor.Api/Innersloth/Customization/PlayerOutfit.cs

```diff
@@ -59,15 +59,15 @@ public string NamePlateId
             internal set => SetField(ref _namePlateId, value);
         }
 
-        private byte HatSequenceId { get; set; } = 0;
+        public byte HatSequenceId { get; set; } = 0;
```

Comment:
Is there a good reason to expose this to the public API? Can we keep this internal to Impostor?

Users shouldn't care about the sequence ID because Impostor should send the next sequence ID